frontend/pages/index.js

```diff
@@ -41,27 +41,27 @@ const Home = () => {
    * @param ModalValue -The current value of the state
    */
   /* eslint-disable no-unused-vars*/
-  const triggerModal = (setModal, ModalValue) => {
+  function triggerModal(setModal, ModalValue) {
```

Comment:
Additionally, your variable name here is `ModalValue` and I've seen it in multiple places where you have inconsistent variable naming.

In javascript, there is a set standard on how to capitalize variables, and in React a specific standard on how to name Components, hooks, etc..

Please examine these best practices and make the appropriate changes across your Repo.

Reply:
Ok, i reviewed my variable, think i got them all named appropriately now